README.md

```diff
@@ -278,6 +278,124 @@ Specifies the phase to be processed.
 > In the case of a call from a subrequest, `auth_jwt_key_request` cannot
 > be processed. (nested in-memory subrequest)
 
+```
+Syntax: auth_jwt_locked_subs_list_file file;
+Default: -
+Context: http, server, location
+```
+
+Specifies a file for locked subs.
```

Comment:
hard to know what it is for

ex: list of JWT sub claims that deny authentication

Reply:
Fixed